examples/amp-video.amp.html

```diff
@@ -83,6 +83,26 @@
     layout="responsive"
     controls>
   </amp-video>
+
+  <h2>Custom Controls</h2>
+  <p>
+     Custom controls replace native controls and can be customized through CSS.
+  </p>
+  <amp-video
+    custom-ctrls
+    custom-ctrls-skin = "light"
+    custom-ctrls-main = "time spacer volume fullscreen"
+    custom-ctrls-mini = "play volume fullscreen"
+    custom-ctrls-floating = "play"
+    title="Big Buck Bunny"
+    artist="Blender Foundation"
```

Comment:
`title` and `poster` are, `artist` and `album` are our own attributes, added as part of the MediaSessionAPI feature